### ISSUE TITLE
tag <head> add attribute like itemscope

### ISSUE THREAD

**Issue by chetanmajithiya28** [nuxt/nuxt.js#4468]
```
headAttrs: {
      'itemscope': ''
    }
        AND
headAttrs: {
      itemscope: ''
    }
```

--> it's not working anyone help please
--> it will not changes but body's and other attributes working

<div align="right"><sub><em>This question is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c8261">#c8261</a>)</em></sub></div>

**Comment by ricardogobbosouza** [comment]
Hi @chetanmajithiya28 

This option does not really exist.

I created a PR https://github.com/nuxt/nuxt.js/pull/4536#issue-238007827

**Comment by manniL** [comment]
Closed by #4536
